src/Native/Unix/System.Native/pal_process.c

```diff
@@ -154,6 +161,9 @@ int32_t SystemNative_ForkAndExecProcess(const char* filename,
                                       int32_t* stdoutFd,
                                       int32_t* stderrFd)
 {
+#ifndef HAVE_PIPE2
```

Comment:
This should be `#if !HAVE_PIPE2` (similar other places). It is the style used everywhere else in this project.

Reply:
Ah; that's actually not a style issue but me completely misreading it. See the Mac build failed. I'm gonna fix this.